great_expectations/datasource/simple_sqlalchemy_datasource.py

```diff
@@ -21,6 +21,7 @@ def __init__(
         engine=None,  # sqlalchemy.engine.Engine
         introspection: dict = None,
         tables: dict = None,
+        **kwargs
```

Comment:
What's the reason for this? Is this to account for `credentials_info`? If so, could we add that as a keyword arg instead?

I know we use the `**kwargs` pattern quite a bit so I understand if it is necessary here.

Reply:
'credentials_info' is an optional argument that applies just to 1 of 2 ways to authenticate for one dialect: bigquery. I worry that making each possible param as named parameter would get bloated quickly

Reply:
good to know! closing my comment